### PR TITLE
Add support for the v1 full text search endpoint

### PR DESCRIPTION
**Pull request by plamenxpenchev**
**Expected approvals: 2**

### CODE REVIEW

**Review by ivaylomitrev — APPROVED** [review]

**Comment by plamenxpenchev** [comment]
Simply rebased after Sign Off support merge, no changes introduced.

File: src/main/java/com/documaster/rms/noark/ws/client/fulltextsearch/FullTextRequest.java
Comment (dkumanov) [inline src/main/java/com/documaster/rms/noark/ws/client/fulltextsearch/FullTextRequest.java]: The doctype could possibly be an enum of the possible values we support.

**Review by dkumanov — APPROVED** [review]

**Comment by ivaylomitrev** [comment]
You still have my approval after the enum change.

File: src/main/java/com/documaster/rms/noark/ws/client/fulltextsearch/FullTextRequest.java
Comment (plamenxpenchev) [reply src/main/java/com/documaster/rms/noark/ws/client/fulltextsearch/FullTextRequest.java]: Addressed with https://github.com/documaster/ws-client-n5-java/pull/33/commits/c81676523ff5005b53b30f622728181e67cfb2cf.

**Comment by dkumanov** [comment]
My approval is still valid as well :green_circle:

**Pull request merged**
